### DISCUSSION TITLE
Unsatabilty of the last version

### DISCUSSION THREAD

**Discussion by Lonelystar52** [Stellarium/stellarium#3118] [General]
Hi everybody,
I us Stellarium for many years with pleasure. 
I downloaded the last Windows version ( 1.22.5 based on Qt 6.4.0).
But now the application closes very often abruptly without warning.
I join the parameters of my computer (sorry it is in french).
Thank you in advance for your help.
Best regards
Patrick Trochet
[PC_Parameters.pdf](https://github.com/Stellarium/stellarium/files/11031606/PC_Parameters.pdf)

**Comment by ilkant** [comment]
Can you explain as accurately as possible what you did when the program crashed? Even one such occasion could help the developers to locate the cause of the error. Example: You tried to perform an action and the program crashed.

**Comment by bruteheru** [comment]
Mine is doing the same thing. windows 11pro. Start program and shuts down when switching between windows on my laptop. If i am in the middle of an alignment and switch to camera, when i come back after alignment it closes as soon as you go back to stellarium. Have not completed a single alignment all the way through most ive gotten is 3 points before shutdown.

**Reply by gzotti** [reply]
Sounds like #2821. It is always good to say what you are doing with the program. A temporary workaround has been identified when running into issues around the TelescopeControl plugin: Use a Qt5-based build.

**Comment by Lonelystar52** [comment]
I use Stellarium connected to my mount (Losmandy Titan 50) to point the object I want to take a picture of.
To make the picture I use Sharpcap or APT.
So I point the target using Stellarium the I run the captures. And when the session is overI dont find Stellarium It has crashed.
Or when i am making  the setup for my session the software crashes.

Best regards
Patrick trochet

**Reply by gzotti** [reply]
Again sounds like #2821. The first message had no indication of any telescope use, leaving us guessing what you were doing.
